packages/mask/src/plugins/Predict/Azuro/components/Teams.tsx

```diff
@@ -0,0 +1,58 @@
+import { Grid, Typography } from '@mui/material'
+import { makeStyles } from '@masknet/theme'
+import { useI18N } from '../../locales'
+import { VersusIcon } from '../icons/VersusIcon'
+import type { Participant } from '../types.js'
+
+const useStyles = makeStyles()((theme) => ({
+    container: { marginBottom: theme.spacing(2.1) },
+    teamContainer: {
+        width: 50,
+        textAlign: 'center',
+        flex: '1',
+    },
+    emblem: {
+        height: 50,
+    },
+    name: {
+        fontSize: 12,
+        fontWeight: 500,
+    },
+    versusIcon: {
+        fill: theme.palette.mode === 'light' ? theme.palette.common.black : theme.palette.common.white,
+    },
+}))
+
+interface TeamsProps {
+    participants: Participant[]
+}
+
+export function Teams(props: TeamsProps) {
+    const t = useI18N()
+    const { participants } = props
+    const { classes } = useStyles()
+
+    return (
+        <Grid className={classes.container} container justifyContent="space-around" wrap="nowrap" alignItems="baseline">
+            <Grid className={classes.teamContainer}>
+                <img
+                    className={classes.emblem}
+                    src={new URL(participants[0].image, import.meta.url).toString()}
+                    alt={t.plugin_home_team()}
+                />
+                <Typography className={classes.name}>{participants[0].name}</Typography>
+            </Grid>
+            <Grid flex="1" textAlign="center">
+                <VersusIcon className={classes.versusIcon} />
+            </Grid>
+            <Grid className={classes.teamContainer}>
+                <img
+                    className={classes.emblem}
+                    src={new URL(participants[1].image, import.meta.url).toString()}
+                    alt={t.plugin_away_team()}
+                />
```

Comment:
The `<AssetPreviewer />` can overcome more network restrictions than `<img />`.

Reply:
I would like to use it but it provoques freezes on the entier plugin window and also add a default mask icon that is not removable. Better to use img for now.